tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -155,6 +155,7 @@
 	defined(CONFIG_BOARD_NUCLEO_L152RE) || \
 	defined(CONFIG_BOARD_OLIMEX_STM32_H103) || \
 	defined(CONFIG_BOARD_96B_AEROCORE2) || \
+	defined(CONFIG_BOARD_STM32F103_MINI) || \
```

Comment:
Please move this change in a dedicated commit

Reply:
No problem.